netlabel/labels.go

```diff
@@ -86,3 +92,23 @@ func Value(label string) string {
 
 	return kv[1]
 }
+
+// KeyValue decomposes the label in the (key,value) pair
+func KeyValue(label string) (string, string, error) {
```

Comment:
Pls reuse `docker/pkg/labels` package to parse the labels instead of reinventing the functionality.